aspnetcore/fundamentals/minimal-apis/includes/webapplication.md

```diff
@@ -0,0 +1,254 @@
+## `WebApplication`
+
+The following code is generated by an ASP.NET Core template:
+
+[!code-csharp[](~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs?name=snippet_default)]
+
+The preceding code can be created via `dotnet new web` on the command line or selecting the Empty Web template in Visual Studio.
+
+The following code creates a <xref:Microsoft.AspNetCore.Builder.WebApplication> (`app`) without explicitly creating a <xref:Microsoft.AspNetCore.Builder.WebApplicationBuilder>:
+
+[!code-csharp[](~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs?name=snippet_create)]
+
+[`WebApplication.Create`](xref:Microsoft.AspNetCore.Builder.WebApplication.Create%2A) initializes a new instance of the <xref:Microsoft.AspNetCore.Builder.WebApplication> class with preconfigured defaults.
+
+### Working with ports
```

Comment:
This seem super niche. How often is it useful the specify the port?  If it's rare, this shouldn't be at the top.